frontend/src/components/SideBarLeft/SideBarLeft.js

```diff
@@ -33,16 +45,33 @@ const SideBarLeft = ({restart, soundOn, isPlaying, instruments, changeCurrentIns
     }
   }
 
+  // handleLineControlToggle takes a line name as an argument and toggles the state of the corresponding line state, e.g. bakerlooControlToggle
+  const handleLineControlToggle = (lineName) => {
+    switch (lineName) {
+      case "Bakerloo":
+        setBakerlooControlToggle(!bakerlooControlToggle);
```

Comment:
very cool, thanks for getting this going. I think itd be ideal to have a little arrow to indicate that the line names are expandable from a UI perspective, as a next step if we have time today